### PR TITLE
Witek/add rewards to employees

### PR DESCRIPTION
**Pull request by piowit**

### CODE REVIEW

**Comment by ciejj** [comment]
@piowit could you please add acceptance criteria to the PR description?

File: spec/system/reward_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+RSpec.describe 'Reward test', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:reward) { create(:reward) }
+
+  it 'test reward' do
+    visit root_path
```
Comment (ciejj) [inline spec/system/reward_spec.rb:14]: Consider using employee login helper here:
https://sajadtorkamani.com/how-to-login-in-capybara-tests-when-using-devise/

**Review by ciejj — APPROVED** [review]
Looks good! :)

**Review by oskarlakner — APPROVED** [review]
🙌

**Pull request merged**